src/components/Input/DropDown/DropDownGroup.js

```diff
@@ -28,39 +29,49 @@ class DropDownGroup extends React.Component {
     document.addEventListener("click", this.handleOutsideClick);
   }
 
+  componentDidUpdate(prevProps, prevState) {
+    const { isOpen } = this.state;
+    // disables scroll when dropdown is open
+    document.addEventListener("wheel", this.disableScroll);
+    // scroll dropdown to top after opening
+    if (isOpen && isOpen !== prevState.isOpen) {
```

Comment:
Can please require that the `isOpen` prop equal false in this conditional?

Reply:
Looks like you made it truthy, rather than falsy

Reply:
I don't know how missed it. Thanks. Fixed now.